docs/data-output.md

```diff
@@ -252,7 +254,7 @@ The weekday cycle is used mainly for weekday readings.
 
 Inside every liturgical year, another psalter week cycle will determine which psalms and prayer to follow for the liturgy of hours.
 
-With the exception of the Easter Octave, each week in the liturgical year is assigned readings from the [Psalter](https://en.wikipedia.org/wiki/Roman_Breviary#The_Psalter).
+Except the Easter Octave, each week in the liturgical year is assigned readings from the [Psalter](https://en.wikipedia.org/wiki/Roman_Breviary#The_Psalter).
```

Comment:
nah, this was a grammatical suggestion

Reply:
Sorry, @TobiTenno, but I still think `for` is required:

According to [this website](https://dictionary.cambridge.org/grammar/british-grammar/except-or-except-for):

- `for` is not required when `except` is after a noun;
- `for` is never used when `except` is used as a conjunction:
  - in that case it seems like we whould be able to insert `that` after `except` when it is used as conjunction, however, it is not the case here.

Sorry to re-open this, but I want to understand this in case I am wrong, or have it fixed if I am right. :pray:

Reply:
I'm not sure what you're reading, because that quite literally says `for` is optional in the cases you're mentioning, or never used when it's a conjunction. As the first word in a subordinate clause, 'for' is not after a noun, and 'for' is not used as a conjunction. Having rules about when a word cannot be used do not directly correlate to all other cases _requiring_ the word. That's not how it works. Once again, in addition to checking it myself, this is the grammar recommended by my IDE.

Reply:
Okay, I get it now, thanks for your patience with me! :pray: